### PR TITLE
Document updated

### PR DESCRIPTION
**Pull request by negarzabetian**
....

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@negarzabetian 
Thanks for your effort ;-)

**Comment by sepandhaghighi** [comment]
@alirezazolanvari @sadrasabouri 
Hi folks ;-)
Please review this PR ;-)

Notice 1 : Do not hurry, we have enough time ;-)
Notice 2 : This PR has 3 reviewer, so please don't merge this PR to `dev` branch before finishing the discussion

Best Regards
Sepand Haghighi

File: README.md [hunk 1]
```diff
@@ -524,7 +524,10 @@ pycm.ConfusionMatrix(classes: ['L1', 'L2', 'L3'])
 
 ### Parameter recommender
 
-This option has been added in `version 1.9` in order to recommend most related parameters considering the characteristics of the input dataset. The characteristics according to which the parameters are suggested are balance/imbalance and binary/multiclass. All suggestions can be categorized into three main groups: imbalanced dataset, binary classification for a balanced dataset, and multi-class classification for a balanced dataset. The recommendation lists have been gathered according to the respective paper of each parameter and the capabilities which had been claimed by the paper.
+This option has been added in `version 1.9` to recommend the most related parameters considering the characteristics of the input dataset. 
+According to the parameters' suggestion, the characteristics are as follows: balance/imbalance and binary/multi-class.
```
Comment (alirezazolanvari) [inline README.md]: This sentence is ambiguous. I suggest to change it as follows. 

The suggested parameters are selected according to some characteristics of the input such as being balance/imbalance and binary/multi-class.

**Comment by sepandhaghighi** [comment]
@negarzabetian Thanks for your quick response, LGTM ;-)
@alirezazolanvari I also made some minor edits, please re-review
@sadrasabouri A friendly reminder

**Pull request merged**